### PR TITLE
FAI-414: Score Cards: Characteristics container should scroll

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.redhat.com/browse/FAI-414

After much fiddling around this is the only solution I found to work. The height can be any value. Absolutely any.

I'd love to know the correct way.. so if you get both the time and urge to look properly it'd be appreciated.

### CODE REVIEW

File: packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CharacteristicsContainer.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@
     background-color: var(--pf-global--BackgroundColor--200);
     height: 100%;
 
+    &-form {
+      height: 0;
+    }
+
```
Comment (kelvah) [inline packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CharacteristicsContainer.scss]: You could change `overview` and `form` like this
```
  &__overview {
    height: 100%;
    display: flex;
    flex-direction: column;
    justify-content: stretch;

    &-form {
      display: block;
      flex: 1 1 0;
      overflow-y: auto;
      padding: 1em;
      background-color: var(--pf-global--BackgroundColor--200);
    }
```

It should be `&__form` following BEM conventions.
You also have to remove [this `section` wrapper ](https://github.com/kiegroup/kogito-tooling/pull/423/files#diff-de9ca0f772724ace94ec55b18f35e1c3813739ad943cd833bab4a3d81459cdd9R109) for it to work.

Comment (manstis) [reply packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CharacteristicsContainer.scss]: Thank-you, thank-you. 

Looks like I need to bring my CSS up to date with Flexbox (nice [site](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) found and bookmarked).

**Comment by manstis** [comment]
@kelvah PR updated with your `Flexbox` improvements. Thank-you.

**Comment by manstis** [comment]
Added @sjamboro and @tiagobento for completeness.

**Review by kelvah — APPROVED** [review]

File: packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CharacteristicsTable.tsx [hunk 2]
```diff
@@ -104,48 +104,47 @@ export const CharacteristicsTable = (props: CharacteristicsTableProps) => {
...
-                />
-              </div>
-            )}
-            {(selectedCharacteristicIndex !== ic.index || activeOperation !== Operation.UPDATE_CHARACTERISTIC) && (
-              <CharacteristicsTableRow
+      {characteristics.map(ic => (
+        <article
+          key={ic.index}
+          className={`editable-item output-item-n${selectedCharacteristicIndex} ${
+            selectedCharacteristicIndex === ic.index && activeOperation === Operation.UPDATE_CHARACTERISTIC
```
Comment (tiagobento) [inline packages/pmml-editor/src/editor/components/EditorScorecard/organisms/CharacteristicsTable.tsx]: Can we extract this boolean expression to a variable? It's being used twice and cluttering the DOM structure. I believe the cleaner the DOM structure is, the better it is to understand the component.

**Comment by manstis** [comment]
@tiagobento Updates made in new commit. Thank-you.

**Comment by manstis** [comment]
Hi @sjamboro could you please try to review some of the PRs?

@tiagobento I assume you're OK with this PR now? Could you please approve it.

**Review by tiagobento — APPROVED** [review]

**Review by sjamboro — APPROVED** [review]

**Comment by sjamboro** [comment]
New issue https://issues.redhat.com/browse/FAI-423.

**Pull request merged**